frontend/src/fetchers/Auth.js

```diff
@@ -0,0 +1,20 @@
+import join from 'url-join';
+
+const callApi = async (url) => {
+  const res = await fetch(url, {
```

Comment:
Have you thought of using `axios` here instead to be consistent between the backend and frontend?

Reply:
We should support IE 11, so either `axios` or a shim for `fetch` is needed. If a shim isn't already in place I'm fine with adding that to the list of things to change after the fact.

Reply:
This work will be done here https://github.com/HHS/Head-Start-TTADP/issues/71